.ibm/pipelines/value_files/values_showcase-rbac.yaml

```diff
@@ -255,9 +257,41 @@ upstream:
         # @default -- `quay.io/janus-idp/backstage-showcase:latest`
         image: '{{ include "backstage.image" . }}'
         command:
-          - python
-          - install-dynamic-plugins.py
-          - /dynamic-plugins-root
+          - sh
+          - '-c'
+          - |
+            cat > /rbac-conditions/conditional-policies.yaml <<EOF
```

Comment:
@josephca Is it possible to move it into a config map and add it in value-files under `extraVolumeMounts` similar to `.ibm/pipelines/resources/config_map/configmap-rbac-policy-rhdh.yaml`

Reply:
@subhashkhileri That's possible. However, if we move it into a config map, it won't allow modifying the conditions on the file by making the file 'readable'.  I intended to generate the condition file to accommodate any upcoming feature improvements and tests. Thanks!